### PR TITLE
List hdf5 as a dependency

### PR DESCRIPTION
**Pull request by daraul**
Possibly related to #2474, this package was a requirement for me to install zipline on arch. I'm not sure that the package name for RHEL and ubuntu is correct, however.

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -44,26 +44,27 @@ binary dependencies from ``apt`` by running:
 
 .. code-block:: bash
 
-   $ sudo apt-get install libatlas-base-dev python-dev gfortran pkg-config libfreetype6-dev
+   $ sudo apt-get install libatlas-base-dev python-dev gfortran pkg-config libfreetype6-dev hdf5
```
Comment (richafrank) [inline docs/source/install.rst]: Is this the right package name? `apt-get install hdf5` fails for me with `Unable to locate package hdf5`.

We've been `apt-get` installing `hdf5-tools`.

File: docs/source/install.rst [hunk 2]
```diff
@@ -44,26 +44,27 @@ binary dependencies from ``apt`` by running:
...
 .. code-block:: bash
 
-   $ pacman -S lapack gcc gcc-fortran pkg-config
+   $ pacman -S lapack gcc gcc-fortran pkg-config hdf5
 
 There are also AUR packages available for installing `ta-lib
 <https://aur.archlinux.org/packages/ta-lib/>`_, an optional Zipline dependency.
 Python 2 is also installable via:
 
+
```
Comment (richafrank) [inline docs/source/install.rst]: Would you update the macOS instructions to include brew installing `hdf5` too?

File: docs/source/install.rst (hunk 1 above)
Comment (daraul) [reply docs/source/install.rst]: Ah, I figured it might be wrong. I'll update.

File: docs/source/install.rst (hunk 2 above)
Comment (daraul) [reply docs/source/install.rst]: Sure!

**Comment by richafrank** [comment]
Thanks for the updates! I also don't know the correct package name for RHEL-derived linux distros, but if it's wrong, we can fix it.

**Pull request merged**